tools/protoc-gen-chpl/message.cpp

```diff
@@ -111,7 +111,7 @@ namespace chapel {
     printer->Print("\n");
 
     printer->Print(
-      "proc serialize(ch) throws {\n"
+      "proc ref serialize(ch) throws {\n"
```

Comment:
I would think that `serialize` wouldn'nt need the `ref` intent here, is that an error or do you know a reason why it is needed?

Reply:
It needed to be `ref` here because of `proc serializeHelper(ref message, ch)` in `modules/packages/ProtobufProtocolSupport.chpl`, which was already `ref`.